### ISSUE TITLE
[HELP NEEDED] KfacOptimizer & cross-entropy loss

### ISSUE THREAD

**Issue by hccho2** [openai/baselines#1095]
classfication loss & kfac

I want to use kfac with cross entropy loss.

1. In this case, how to set fisher loss?
2. the roles of fisher loss?
```
loss = tf.reduce_mean(tf.nn.softmax_cross_entropy_with_logits_v2(logits=logits,labels=Y))

optimizer = kfac.KfacOptimizer(learning_rate=learning_rate, clip_kl=kfac_clip,\
			momentum=0.9, kfac_update=1, epsilon=0.01,\
			stats_decay=0.99, is_async=is_async, cold_iter=10, max_grad_norm=max_grad_norm)


fisher_loss  = ???
optimizer.compute_and_apply_stats(fisher_loss,  var_list=params) 
train_op, q_runner = optimizer.apply_gradients(list(zip(grads,params)))
```
